specification/communication/data-plane/Sms/preview/2024-12-10-preview/communicationservicessms.json

```
@@ -197,6 +329,84 @@
           "type": "string"
         }
       }
+    },
+    "OptOutSendRecipient": {
+      "description": "Represents properties for a single recipient.",
+      "required": [
+        "to"
+      ],
+      "type": "object",
+      "properties": {
+        "to": {
+          "description": "The recipient's phone number (in E.164 format).",
+          "minLength": 1,
+          "type": "string"
+        }
+      }
+    },
+    "OptOutSendRequest": {
+      "description": "Represents properties for a Opt request.",
+      "required": [
+        "from",
+        "recipients"
+      ],
+      "type": "object",
+      "properties": {
+        "from": {
+          "description": "The sender's identifier (typically phone number in E.164 format) that is owned by the authenticated account.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "recipients": {
+          "maxItems": 100,
+          "minItems": 1,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OptOutSendRecipient"
+          }
+        }
+      }
+    },
+    "OptOutSendResponse": {
+      "description": "Response for a successful or multi status Opt send request.",
```

"multi status" implies a 207 response, let's avoid this term and rephrase to something like
```suggestion
      "description": "Response for an opt out request. Validate the returned items in the response to see which recipients were successfully added or removed from the opt outs list.",
```